### DISCUSSION TITLE
ChatCompletionChunk missing usage property.

### DISCUSSION THREAD

**Discussion by jay-pulsifi** [instructor-ai/instructor-js#117] [Q&A]
Hi team,

Would like to request to add usage property in `ChatCompletionChunk` interface similar to `ChatCompletion`.

This would be helpful for our use case where it allow us to track the statistics of each request.

<img width="650" alt="image" src="https://github.com/instructor-ai/instructor-js/assets/107604394/32580742-53a6-4fa6-a788-b3bff05bc795">
<img width="637" alt="image" src="https://github.com/instructor-ai/instructor-js/assets/107604394/ec9ff348-6d03-4617-a9b1-6a1c69f5d3a8">

Thanks in advance

**Comment by jxnl** [comment]
where does this class come from?

**Reply by jay-pulsifi** [reply]
My apologies. I dropped a question on wrong repo.
